src/Integration.Vsix/CFamily/CLangAnalyzer.cs

```diff
@@ -41,18 +41,18 @@ internal class CLangAnalyzer : IAnalyzer
         private readonly ITelemetryManager telemetryManager;
         private readonly ISonarLintSettings settings;
         private readonly ICFamilyRulesConfigProvider cFamilyRulesConfigProvider;
+        private readonly IAnalysisStatusNotifier analysisStatusNotifier;
         private readonly ILogger logger;
         private readonly DTE dte;
 
         [ImportingConstructor]
-        public CLangAnalyzer(ITelemetryManager telemetryManager, ISonarLintSettings settings, ICFamilyRulesConfigProvider cFamilyRulesConfigProvider,
-            [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, ILogger logger)
+        public CLangAnalyzer(ITelemetryManager telemetryManager, ISonarLintSettings settings, ICFamilyRulesConfigProvider cFamilyRulesConfigProvider, [Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider, IAnalysisStatusNotifier analysisStatusNotifier, ILogger logger)
```

Comment:
You don't need to import the service provider here.

Reply:
It was already there, it's used to get the DTE